### ISSUE TITLE
Export Zone

### ISSUE THREAD

**Issue by synw** [NiKoTron/geohex#1]
Hi, cool package. There is an error in the readme: the decode method returns a `Zone` and not a string:

```dart
String geoHexCode = GeoHex.decode('OM4138');
```

should be:

```dart
Zone geoHexCode = GeoHex.decode('OM4138');
```

The `Zone` class should be exported: I want to use the `hexCoords` methods to draw the hexagons on a map. I think the name Zone is a bit too broad: could it be named `GeoHexZone` for more precision and to avoid name clashes?

**Comment by NiKoTron** [comment]
Hi! thank you for interest to the lib and for reporting.

I'll fix the mistake.

About renaming Zone to smth. another, so actually I think that is not a big problem to use prefixes. 

for example:

```dart
import 'package:geohex/geohex.dart' as geohex;
 
geohex.Zone zone;
```

I wanna try to leave names of the public classes as close as possible to the original.

**Comment by synw** [comment]
It is not really a problem to use namespaces but I think having clear and distinct names for classes is a good practice. I faced a namespace issue with a package I am working on ([see here](https://github.com/synw/geojson/issues/4)) and I had the same reaction than you telling the guy to use custom namespaces. 

In the end after using the lib for a while with custom namespacing I realized that directly prefixing the class names is much more convenient than custom namespacing because of the confusion introduced by the names similarity at some point when the code gets bigger: it becomes confusing and creates problems for nothing

> I wanna try to leave names of the public classes as close as possible to the original

I understand this but wouldn't it be worth it to make the names closer to the Dart way? I mean there are more benefits doing this than to keep the original names. Anyway that"s just an opinion and it should not be a big problem in the end, Zone should not be that common
